service/servicelayerfunctionalities.cc

```diff
@@ -320,19 +320,34 @@ Status ServerForCommandLineClient::stream(
   }
 
   std::set<std::string> chirpsent; /* Save all the id of chirps was sent */
-  chirp::Chirp chirp;
-  chirp.ParseFromString(from_get_function[0]);
+  chirp::Hashtag hashtag;
+  hashtag.ParseFromString(from_get_function[0]);
 
   chirp::StreamReply reply;
   /*
     Continuously look through in db for #. Keep all
     sent chirps in a set called chirpsent to disallow sending duplicates. Keep
     looking for new chirps with this while loop.
   */
-  // TODO: Implement continous polling for new chirps from #hashtag
   while (true) {
     if (context->IsCancelled()) {
+      break;
+    }
+    auto from_get_function = client_key.get("hashtag#" + request->hashtag());
+    if (from_get_function.size() == 0) {
+      break;
+    }
+
+    // Go thru all chirps under hashtag and check if they are new
```

Comment:
/* */ instead of //

Reply:
This is a tricky choice; on one hand, `//` is preferred in C++ generally, but on the other the rest of the project seems to not use it, so maybe you'd prefer consistency.

Reply:
ill just be consistent